libs/blocks/mnemonic-list/mnemonic-list.css

```diff
@@ -1,4 +1,4 @@
-.mnemonic-list {
+.marquee .mnemonic-list {
```

Comment:
This does fix the effect, but I'm not certain it's the cause. Looking at the markup, I see `.mnemonic-list` being applied to the `.marquee` element, but is also a standalone element nested inside. From the outside, I'd say that `.mnemonic-list` should not be added as a class to the marquee. Is there a reason behind it? I'd argue that we should keep class names relevant to their purpose and this feels counter-intuitive to me
<img width="641" alt="Screenshot 2024-02-01 at 14 44 43" src="https://github.com/adobecom/milo/assets/11267498/481e9f52-6092-432f-a115-8a3b8af99999">

Reply:
I agree, I'll defer the refactoring to @Axelcureno , this is to fix the immediate CLS issue.

Reply:
This block is made to be used either standalone or inside another block and when using inside a block it requires minimal styling (this may not be standard across blocks). Since Marquee is the first block using it, we are simply introducing styles to match the XD specs of the usage. We can remove it for now and make the styles standard, but it's likely that we will need to re introduce this later. For example, Merch Card will use this block soon.